modules/nodes-descheduler-configuring-strategies.adoc

```diff
@@ -0,0 +1,56 @@
+// Module included in the following assemblies:
+//
+// * nodes/scheduling/nodes-descheduler.adoc
+
+[id="nodes-descheduler-configuring-strategies_{context}"]
+= Configuring descheduler strategies
+
+You can configure which strategies that the descheduler uses to evict Pods.
+
+.Prerequisites
+* Cluster administrator privileges.
+
+.Procedure
+
+. Edit the `KubeDescheduler` object:
++
+----
+$ oc edit kubedeschedulers.operator.openshift.io cluster -n openshift-kube-descheduler-operator
+----
+
+. Specify one or more strategies in the `strategies` section of the `spec` section.
```

Comment:
I prefer "dot" notation, but I don't know if we have a clear guidance.
_...in the spec.strategies section._

Reply:
This is all set, you already Slacked me, lol.